lesson-3-implementing-message-passing/flask-demo/app.py

```diff
@@ -7,15 +7,15 @@
 
 @app.route('/')
 def hello_world():
-    return 'Hello World!'
+    return jsonify({'message': 'Hello World!'})
 
 @app.route('/demo/<path_demo>', methods=['POST'])
 def demo(path_demo=None):
     # Showcase different ways in which data can be passed
     path_value = path_demo
     header_value = request.headers.get('header_demo', None)
     param_value = request.args.get('param_demo', None)
-    body_value = request.data
+    body_value = request.data.decode('UTF-8')
```

Comment:
Without that python is printing the string with a leading `b'` to indicate, that it's binary. Confusing to new users. See https://stackoverflow.com/questions/57337321/flask-b-text-appears-before-request-data-results.